README.md

```diff
@@ -257,6 +257,13 @@ With the newly arrived data, BayBE can produce a refined design for the next ite
 This loop would typically continue until a desired target value has been achieved in
 the experiment.
 
+## Authors
+

```

Comment:
we have no consistent behavior os either having or not having a newline after headlines, i dont like them but it seems others like them, lets just decide on one particular version and not be inconsistent

Reply:
Agree. Should however discuss at a later point since we should then have a look at ALL .md files, and in particular need to check if having newlines is e.g. critical for the conversion of examples.